node/metered-channel/src/lib.rs

```diff
@@ -107,8 +108,10 @@ impl Meter {
 /// Determine if this instance shall be measured
 #[inline(always)]
 fn measure_tof_check(nth: usize) -> bool {
-	const TOF_MEASURE_EVERY_NTH: usize = if cfg!(test) { 2 } else { 16 };
-	((TOF_MEASURE_EVERY_NTH - 1_usize) & nth) == 0
+	let mut rng = rand::thread_rng();
+	let val = rng.gen_range(0..1000);
+	const TOF_MEASURE_EVERY_NTH: usize = if cfg!(test) { 2 } else { 17 };
+	val == 0 && ((TOF_MEASURE_EVERY_NTH - 1_usize) & nth) == 0
```

Comment:
This cheak doesn't work anymore, it assumed `TOF_MEASURE_EVERY_NTH` was or the form `pow(2,x)`. I'll fix this real quick